src/main/scala/com/sksamuel/scapegoat/sbt/ScapegoatSbtPlugin.scala

```diff
@@ -116,11 +116,11 @@ object ScapegoatSbtPlugin extends AutoPlugin {
     } ++ Seq(
       (compile in Scapegoat) := ((compile in Scapegoat) dependsOn scapegoatClean).value,
       scapegoat := (compile in Scapegoat).value,
-      scapegoatCleanTask := doScapegoatClean((scapegoatRunAlways in ThisBuild).value, (classDirectory in Scapegoat).value, streams.value.log),
+      scapegoatCleanTask := doScapegoatClean((scapegoatRunAlways in ThisProject).value, (classDirectory in Scapegoat).value, streams.value.log),
```

Comment:
I wonder if we actually need to be explicit with scopes here. Thinking if it would be better to just let sbt follow the scope hierarchy by itself, rather than being explicit where it should start.